core/src/main/java/org/infinispan/remoting/transport/jgroups/EmbeddedJGroupsChannelConfigurator.java

```diff
@@ -59,7 +59,7 @@ public String getProtocolStackString() {
 
    @Override
    public List<ProtocolConfiguration> getProtocolStack() {
-      return combineStack(jgroupsConfiguration.configurator(parent), stack);
+      return combineStack(parent == null ? null : jgroupsConfiguration.configurator(parent), stack);
```

Comment:
Maybe it would have been better to add the null check in `JGroupsConfiguration.configurator(String name)`